tests/integration/bond_test.py

```diff
@@ -190,7 +190,7 @@ def test_add_bond_with_port_and_ipv4(eth1_up, eth2_up, setup_remove_bond99):
                         eth1_up[Interface.KEY][0][Interface.NAME],
                         eth2_up[Interface.KEY][0][Interface.NAME],
                     ],
-                    Bond.OPTIONS_SUBTREE: {"miimon": "140"},
+                    Bond.OPTIONS_SUBTREE: {"miimon": 140},
```

Comment:
This is incorrect. We should change what we already supported.

A better approach might be using `HashMap<String, serde_json::Value>` instead of `struct BondOptions`.

Reply:
I have follow a different approach. I think it is important to be able to control all the supported properties. I have introduced a custom deserializing function `json_to_u32`, `json_to_u16` and `json_to_u8`. This way, the bond options will accept `"140"` and `140`. Thanks!